src/main/java/uk/gov/hmcts/ccd/domain/service/createcase/SubmitCaseTransaction.java

```diff
@@ -28,10 +20,18 @@
 import uk.gov.hmcts.ccd.domain.service.common.UIDService;
 import uk.gov.hmcts.ccd.domain.service.stdapi.AboutToSubmitCallbackResponse;
 import uk.gov.hmcts.ccd.domain.service.stdapi.CallbackInvoker;
-import uk.gov.hmcts.ccd.endpoint.exceptions.CaseConcurrencyException;
+import uk.gov.hmcts.ccd.endpoint.exceptions.ReferenceKeyConcurrencyException;
 import uk.gov.hmcts.ccd.infrastructure.user.UserAuthorisation;
 import uk.gov.hmcts.ccd.infrastructure.user.UserAuthorisation.AccessLevel;
 
+import javax.inject.Inject;
+import javax.transaction.Transactional;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+
+import static javax.transaction.Transactional.TxType.REQUIRES_NEW;
+import static uk.gov.hmcts.ccd.data.caseaccess.GlobalCaseRole.CREATOR;
+
```

Comment:
imports seems to have changed in order. could we make sure we're using the standard code format activated to be applied on auto-save action?

Reply:
Imports are fine. Ordered by the formatter.